src/pages/events.js

```diff
@@ -157,7 +157,16 @@ class Events extends Component {
       <Consumer>
         {context => (
           <PageWrapper>
-            <Helmet title="What's on - LGBT+ events happening in and around London" />
+            <Helmet
```

Comment:
This looks like the SEO-9 ticket (`/events`). Could you pop this in a separate PR so that it's linked to the right ticket in Jira?

Reply:
Sure